### ISSUE TITLE
Remove Quandl

### ISSUE THREAD

**Issue by optwave** [cvxgrp/cvxportfolio#19]
@enzobusseti Quandl is used within HelloWorld.ipynb, RealTimeOptimization.ipynb and DataEstimatesRiskModel.ipynb 
Quandl performs communication with a remote server whose internals cannot be inspected and audited  (essential properties for financial management software). I proposed you a powerful addon for hyperparameter optimization which consists of about 100 lines of open source code  to make API requests to a remote solver (Quandl performs similar requests to its remote server). You classified it as a closed code and mentioned that inserting components with closed source  is contrary to the philosophy of this work, make it less attractive to users, and less trustworthy. Therefore, in order to be consistent with your previous assessment, Quandl should be removed.

**Comment by enzobusseti** [comment]
hi! `quandl` is included for convenience but it's only used in the examples, `cvxportfolio` does not depend on it. you don't have to use it!

**Comment by optwave** [comment]
The same was true for the extension that I proposed, i.e., an advanced hyperparameter optimizer could be conveniently used (i.e., you don't have to use it and `cvxportfolio` would not depend on it) on top of  random search and grid search which are inefficient for large (>3) number of hyperparameters. Yet, in the same situation you classified my proposed open source caller and optimization code as a closed code insertion which can make `cvxportfolio` less attractive to users. 
I don't understand your policy/preference model for giving green light to some contributions and red light the other contributions of the same kind. However, I believe that it should be consistent, therefore, I opened this ticket.
